docs/intelligentapps/tracing.md

```diff
@@ -630,6 +798,375 @@ Use the following instructions to deploy a preconfigured development environment
 
     ![Screenshot showing the Trace Details view in the Tracing webview.](./images/tracing/trace_details.png)
 
+## Example 2: set up tracing with the Azure AI Inference SDK using Monocle
```

Comment:
I think only one example to show the experience is good enough and there is no need to include three examples. @MuyangAmigo What's your opinion?